PublishParasoftTestResults/package.json

```diff
@@ -4,7 +4,12 @@
   "description": "Azure Pipelines Publish Parasoft Test Results Task",
   "main": "publishtestresults.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "compilexsl": "(npm run sarifxsl) && (npm run xunitxsl) && (npm run soatestxsl) ",
+    "sarifxsl": "node node_modules/xslt3/xslt3.js -t -xsl:./xsl/sarif.xsl -export:./xsl/sarif.sef.json -nogo",
+    "xunitxsl": "node node_modules/xslt3/xslt3.js -t -xsl:./xsl/xunit.xsl -export:./xsl/xunit.sef.json -nogo",
+    "soatestxsl": "node node_modules/xslt3/xslt3.js -t -xsl:./xsl/soatest-xunit.xsl -export:./xsl/soatest-xunit.sef.json -nogo",
+    "licenses": "./node_modules/.bin/license-checker-rseidelsohn --excludePackages \"parasoft-findings-vsts-publish\" --limitAttributes \"licenses,licenseFile,publisher,repository\" --files licenses --relativeLicensePath --json --out licenses/license.json",
```

Comment:
This doesn't work for me when I run "npm run licenses".
I got "No such file or directory" error.
You probably need to put "bash -c" in front of the command line to make sure it runs as a command.

Reply:
@rwang-parasoft I believe we should include this into "npm install" process. What do you think?

Reply:
> This doesn't work for me when I run "npm run licenses". I got "No such file or directory" error. You probably need to put "bash -c" in front of the command line to make sure it runs as a command.

I changed the command line and verified it works with *Windows PowerShell*, *Command Prompt*, *MSYS2 Terminal* and *Cygwin* on my environment and also on a Ubuntu VM, you can double-check it on you environment.

Reply:
> > This doesn't work for me when I run "npm run licenses". I got "No such file or directory" error. You probably need to put "bash -c" in front of the command line to make sure it runs as a command.
> 
> I changed the command line and verified it works with _Windows PowerShell_, _Command Prompt_, _MSYS2 Terminal_ and _Cygwin_ on my environment and also on a Ubuntu VM, you can double-check it on you environment.

Looks like 

> > This doesn't work for me when I run "npm run licenses". I got "No such file or directory" error. You probably need to put "bash -c" in front of the command line to make sure it runs as a command.
> 
> I changed the command line and verified it works with _Windows PowerShell_, _Command Prompt_, _MSYS2 Terminal_ and _Cygwin_ on my environment and also on a Ubuntu VM, you can double-check it on you environment.

Works for me

Reply:
I read online and browsed some examples, and it seems like a convention that the license folder should reside in the root directory. Can we generate the license folder in the root directory and create a subfolder for "PublishParasoftTestResults"? This would be well-structured in case we add more features to the extension in the future, and there are more than one task folders.